src/components/Form/SetAclPermissionForm.jsx

```diff
@@ -59,37 +55,23 @@ const SetAclPermissionForm = () => {
     try {
       await setDocAclPermission(session, docType, permissions, podUrl, webIdToSetPermsTo);
 
-      runNotification(
-        `${permissions.read ? 'Give' : 'Revoke'} permission to ${
-          permissionState.podUrlToSetPermissionsTo
-        } for ${docType}.`,
-        5,
-        state,
-        dispatch
-      );
       addNotification(
         'success',
-        `${permissions.read ? 'Gave' : 'Revoked'} permission to ${
+        `${permissions.read ? 'Gave permission to' : 'Revoked permission from'} ${
           permissionState.podUrlToSetPermissionsTo
         } for ${docType}.`
       );
     } catch (error) {
-      runNotification('Failed to set permissions. Reason: File not found.', 5, state, dispatch);
       addNotification('error', 'Failed to set permissions. Reason: File not found.');
     } finally {
       setTimeout(() => {
```

Comment:
We can still remove the `setTimeout` though. No reason to wait 3 seconds to clear processing if processing is finished

Reply:
I can remove the setTimouts in these modals, and the modals will go away immediately instead of the two second delay it currently has. If we do that there will be no need for the setProcessing state because the button will be instantly gone as well.

Reply:
No, we'll still need the `setProcessing` state to disable the button while it's running to prevent duplicate event calls.

Reply:
Maybe we can do something like this:

<img width="1004" alt="Screenshot 2023-08-23 at 8 45 25 AM" src="https://github.com/codeforpdx/PASS/assets/71395931/09bb8182-4a02-4dc1-b665-16c2f0ab5804">

That way if the async function takes a while to process for whatever reason - the button to run it again is disabled. Then modal disappears after receiving a Resolve or Reject.

?

Reply:
Maybe this whole discussion would be faster as a quick call.   I'm still not sure exactly when we want the modals to disappear.

If we are going to use some sort of animation in the modal itself (loading button was what I proposed) to show user that something is happening, then we'd want it to disappear after the task is complete.

If we are going to use a separate component to show user that something is happening, then we'd want it to disappear immediately after button press.


So, the answer to this effects my new PR work, and the next one to make all the modals more consistent in styling/look.